net-mgmt/zabbix-agent/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		zabbix-agent
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		Enterprise-class open source distributed monitoring agent
+PLUGIN_DEPENDS=     zabbix32-agent
```

Comment:
There's no point using such an outdated version anymore. I strongly suggest to use 3.2 (or at a minimum 3.0) by now.